content/dogepedia/articles/dogecoin-roadmap.md

```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-Dogecoin and roadmaps.
+While a 'roadmap' is a term that's commonly used when new cryptocurrencies are released, it also conveys a rigid way forward, led by a single ‘company’ or ‘lead developer’.  
+
+Dogecoin is a community-driven, consensus based open-source project and its direction is formed by the many individual and organisational contributors to the various projects of the Dogecoin blockchain & ecosystem.  
+
+The direction of Dogecoin is paved by the [many projects](https://foundation.dogecoin.com/trailmap/) that the community feels best shapes its direction. This is a very intentional decision: it is important that Dogecoin remains a decentralised project with shepherds rather than rulers, each whose projects and contributions stand or fall on their merits.  

```

Comment:
Possibly some confusion here, maybe speaking about Dogecoin Core instead of Dogecoin can be more accurate. Also, development is done in https://github.com/dogecoin/dogecoin, using also GitHub roadmaps : https://github.com/dogecoin/dogecoin/projects to plan work needed for next versions :)

It must speak about the GitHub repo, some improvement need to be done to limit confusion between foundation trailmap and software development "roadmap". The foundation trailmap contain a lot of projects around Dogecoin Core, not directly related to the Dogecoin Core software development.

Of course it's still a draft, but I invite you to take care of this explanation, confusion between Foundation roadmap and Dogecoin Core roadmap can be a sensitive point in the community, clarity will be important here :heart: !

Reply:
I agree with @AbcSxyZ, these are good points that should be included also in the first version of the article. I will try to propose some changes in the next days!